### PR TITLE
Add lane line and mini car for toyota dash display

### PR DESCRIPTION
**Pull request by arne182**

### CODE REVIEW

**Comment by arne182** [comment]
Merry Christmas

File: selfdrive/car/toyota/carcontroller.py [hunk 1]
```diff
@@ -213,13 +213,16 @@ def update(self, sendcan, enabled, CS, frame, actuators,
                                                  ECU.APGS in self.fake_ecus))
     elif ECU.APGS in self.fake_ecus:
       can_sends.append(create_ipas_steer_command(self.packer, 0, 0, True))
-
+    if lead or CS.v_ego < 11.2: # cannot be on below 40 kph
```
Comment (rbiasini) [inline selfdrive/car/toyota/carcontroller.py]: what is `mini_car`?

File: selfdrive/car/toyota/carcontroller.py [hunk 2]
```diff
@@ -239,7 +242,7 @@ def update(self, sendcan, enabled, CS, frame, actuators,
       send_ui = False
 
     if (frame % 100 == 0 or send_ui) and ECU.CAM in self.fake_ecus:
-      can_sends.append(create_ui_command(self.packer, steer, sound1, sound2))
+      can_sends.append(create_ui_command(self.packer, steer, sound1, sound2, 0, 2 - left_line, 2 - right_line))
```
Comment (rbiasini) [inline selfdrive/car/toyota/carcontroller.py]: why sending `2 - xxx_line` and not simply 'xxx_line'?

Comment (arne182) [reply selfdrive/car/toyota/carcontroller.py]: 0 is for lane detection completely off. 1 and 2 are for solid line and line not seen. But from the logic it is reversed. Having it only send xxx_line shows off when it skills show on.

Comment (rbiasini) [reply selfdrive/car/toyota/carcontroller.py]: ah ok. So makes sense to send 2 when line is not detected.

File: selfdrive/car/toyota/carcontroller.py (hunk 1 above)
Comment (rbiasini) [reply selfdrive/car/toyota/carcontroller.py]: it seems like you are forced to turn it off below 40kph. Isn't that confusing? What happens if you keep it on at lower speeds?

Comment (arne182) [reply selfdrive/car/toyota/carcontroller.py]: Confusing, yes, but better than always on all the time

**Comment by arne182** [comment]
Do you you want these two comments in the code?

File: selfdrive/car/toyota/carcontroller.py (hunk 1 above)
Comment (rbiasini) [reply selfdrive/car/toyota/carcontroller.py]: should 11.2 be replaced by minSetSpeed?

Comment (arne182) [reply selfdrive/car/toyota/carcontroller.py]: That is a very good idea.

**Comment by rbiasini** [comment]
I copied this changes on the internal branch with some other minor clean up. It will be in next release with mention. Thanks!

**Pull request closed without merging**